### DISCUSSION TITLE
Блокировка?!

### DISCUSSION THREAD

**Discussion by romanlipov** [Chia-Network/chia-blockchain#6000] [Support]
здравствуйте! у меня появилась блокировка! не понимаю что это такое! сею на двух фермах только в соло! я сильно волнуюсь, возможно из за маленького опыта сделал что то неправильно?! что мне делать с блокировкой? и что она значит? помогите пожалуйста!
мой ip: 192.168.1.244
![IMG_20210525_110911](https://user-images.githubusercontent.com/84781248/119499211-297cdb00-bd6f-11eb-8728-0adc7931159a.jpg)

**Comment by romanlipov** [comment]
Может блокировка из за крана Chia?
![IMG_20210525_122648](https://user-images.githubusercontent.com/84781248/119500161-28987900-bd70-11eb-8120-01ffa28d4abc.jpg)

**Comment by BrandtH22** [comment]
Hey @romanlipov , these are sites that CNI does not control (keybase and chia.toga.....) I recommend reaching out to the companies that manage those sites for more information.
